### PR TITLE
Add Availabilities

### PR DESCRIPTION
**Pull request by agrare**
Depends on:

- [x] https://github.com/ManageIQ/topological_inventory-core/pull/98/files

### CODE REVIEW

File: public/doc/swagger-2-v0.0.1.yaml [hunk 1]
```diff
@@ -837,6 +837,20 @@ paths:
           description: Source deleted
         404:
           description: Not found
+  "/sources/{id}/availabilities":
```
Comment (bdunne) [inline public/doc/swagger-2-v0.0.1.yaml]: Do we need to add this to the v0.0 api?

Comment (agrare) [reply public/doc/swagger-2-v0.0.1.yaml]: Probably not

Comment (agrare) [reply public/doc/swagger-2-v0.0.1.yaml]: Removed

File: public/doc/swagger-2-v0.1.0.yaml [hunk 2]
```diff
@@ -2200,9 +2276,6 @@ definitions:
         readOnly: true
         title: Name
         type: string
-      resource_version:
```
Comment (bdunne) [inline public/doc/swagger-2-v0.1.0.yaml]: This shouldn't be removed

Comment (agrare) [reply public/doc/swagger-2-v0.1.0.yaml]: Hm I assumed this was missing from the core models but it is there on ServicePlans, need to look at why the generator cleared this.

Comment (agrare) [reply public/doc/swagger-2-v0.1.0.yaml]: Fixed

Comment (bdunne) [reply public/doc/swagger-2-v0.1.0.yaml]: Maybe you didn't have the latest version of `topological_inventory-core` when the generator was run?

**Pull request merged**